DXRModules/DeusEx/Classes/DXRMissions.uc

```diff
@@ -1016,6 +1214,25 @@ function MoveGoalToLocation(Goal g, GoalLocation Loc)
             m.Tag = '';
             m.Event = '';
         }
+    } else if (g.name=="Area 51 Blast Door Computer" && Loc.name != "the tower") {
+        foreach AllActors(class'#var(prefix)DataCube',dc1){
+            if(dc1.TextTag=='15_Datacube07'){
+#ifdef injections
+                dc2 = Spawn(class'#var(prefix)DataCube',,, dc1.Location, dc1.Rotation);
```

Comment:
does this run before or after DXRReplaceActors turns them into DXRInformationDevices? will need to check this